slsim/Sources/SourceVariability/light_curve_interpolation.py

```diff
@@ -0,0 +1,27 @@
+from scipy.interpolate import interp1d
+
+
+class LightCurveInterpolation(object):
+    """This class manages interpolation of light curve of a source."""
+
+    def __init__(self, times, magnitudes):
+        """
+        :param times: array of observation time
+        :param magnitudes: array of magnitude
+        """
+        self.time_array = times
+        self.magnitude_array = magnitudes
+        self.interpolation = interp1d(
+            self.time_array,
+            self.magnitude_array,
+            kind="linear",
+            fill_value="extrapolate",
```

Comment:
is 'extrapolate' good or do we want to set all outside values to zero?

Reply:
I think setting all outside values to zero is a good idea.